cmd/juju/storage/volumelist.go

```diff
@@ -89,15 +104,3 @@ func (c *VolumeListCommand) Run(ctx *cmd.Context) (err error) {
 	}
 	return c.out.Write(ctx, output)
 }
-
-var getVolumeListAPI = (*VolumeListCommand).getVolumeListAPI
-
-// VolumeListAPI defines the API methods that the volume list command use.
-type VolumeListAPI interface {
-	Close() error
-	ListVolumes(machines []string) ([]params.VolumeDetailsResult, error)
-}
-
-func (c *VolumeListCommand) getVolumeListAPI() (VolumeListAPI, error) {
```

Comment:
Having said this, I see that your changed approach is actually consistent with the way that other commands do it. Consistency wins, I think, but do what you feel is right (and minimises changes in this PR)